src/Treemap.js

```diff
@@ -23,6 +23,7 @@ export default class Treemap extends Viz {
 
     this._layoutPadding = 1;
     this._shapeConfig = assign({}, this._shapeConfig, {
+      ariaLabel: (d, i) => `${d.index + 1}. ${this._drawLabel(d, i)}, ${d.value}.`,
```

Comment:
We cannot use `d.value` directly here, as the user may have defined their own accessor. This should be `this._sum(d, i)`